acceptance-tests/src/test/java/tech/pegasys/signers/dsl/hashicorp/HashicorpVaultDocker.java

```diff
@@ -177,20 +165,32 @@ private void postStartup() {
     hashicorpRootToken = hashicorpVaultTokens.getRootToken();
   }
 
-  private void createSecretKey() {
+  public String addSecretsToVault(final Map<String, String> entries, final String path) {
```

Comment:
The method is doing two things, isn't it :).

Reply:
as in, inserting the key, and returning the path?

Reply:
yeah ... feels a bit off tbh (or may be I am overthinking). You already have a separate method to construct the "path" from "/secret". Call that whereever the "path" is required and this method can remain void? not-a-show-stopper. This is just nit-picking.